extensions/opencv/CMakeLists.txt

```diff
@@ -32,6 +32,10 @@ add_library(minifi-opencv SHARED ${SOURCES})
 
 target_link_libraries(minifi-opencv ${LIBMINIFI})
 target_link_libraries(minifi-opencv OPENCV::libopencv)
+if (APPLE)
+    target_link_libraries(minifi-opencv "-framework AVFoundation" "-framework CoreFoundation" "-framework CoreGraphics" "-framework CoreMedia" "-framework CoreVideo" "-framework Foundation" "-framework OpenCL")
+    target_link_libraries(minifi-opencv -llapack -lblas)
```

Comment:
Are these the dependencies of OpenCV itself? If so, it should be handled on the OpenCV lib target. Ideally the OpenCV project itself would link these to itself, or advertise them as required.

Reply:
Good point, I've moved these into BundledOpenCV instead. https://github.com/apache/nifi-minifi-cpp/pull/1694/commits/56aa70c970d2c68ce02cac0688bd62bea4a43bd6

For the requirement of blas and lapack, it does seem opencv requires them both
But after reading through opencv macOS related issues it seems its recommended to use the Accelerate framework instead of manually linking these libs.
https://github.com/apache/nifi-minifi-cpp/pull/1694/commits/834c5f438fed1804320b3c211681014c6f9499e8